### PR TITLE
New layerpanel design

### PR DESCRIPTION
**Pull request by drodenberg**
Resolves: attach_gh_issue_here

### PR Safety Checklist:

 - [x] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [ ] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [x] (optional) All external API changes have been documented
 - [x] (optional) Build docs: `npm run docs`

### CODE REVIEW

**Comment by drodenberg** [comment]
Screen Shot 2020-10-14 at 3 56 10 PM

**Comment by drodenberg** [comment]
Screen Shot 2020-10-14 at 3 56 03 PM

**Comment by drodenberg** [comment]
Screen Shot 2020-10-14 at 4 21 01 PM

File: app/tests/LayerPanel.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 const puppeteer = require('puppeteer')
 const assert = require('assert')
-const { assertWithInfo } = require('./pupp-utils')
+const { assertWithInfo, addDebugCSS } = require('./pupp-utils')
 
 ;(async () => {
   const browser = await puppeteer.launch({
-    headless: true,
+    headless: false,
```
Comment (drodenberg) [inline app/tests/LayerPanel.js:7]: flip this to true

File: src/LayerPanel/LayerPanelBase/index.js [hunk 2]
```diff
@@ -16,43 +18,46 @@ class LayerPanelBase extends Component {
...
-            })
-          }
-        </Tabs>
-        {React.Children.toArray(children)[activeIndex - 1]}
-      </Card>
+      <>
+        {!showLayerPanel && <InitialTab id='initialtab' onClick={this.showLayerPanel} icon={<LayersIcon data-testid={tabDataTestId} />} />}
+        <Card open={showLayerPanel} styles={style} numoftabs={children.length || 1} inline={inline} className='_popup_boundary' >
+          <CardContent>
+            <Typography variant='h5' component='h5'>Layer Panel - Map 1</Typography>
```
Comment (drodenberg) [inline src/LayerPanel/LayerPanelBase/index.js:48]: `layerPanelTitle` prop for the typography here

**Comment by drodenberg** [comment]
Try and drop the drag component over this and also give it some padding above the title

**Comment by glenselle** [comment]
We can try drag and drop in another PR -- merging ✅

**Review by glenselle — APPROVED** [review]

**Pull request merged**